doc/kubeletConfig-check.md

```diff
@@ -0,0 +1,141 @@
+# KubeletConfig default checks usage
+
+This page describes the usage of the `KubeletConfig` default checks in the compliance operator.
+`KubeletConfig` rules are being changed from Node checks to Platform checks. Compliance Operator
+is able to check runtime `KubeletConfig` through `Node/Proxy` object, and it is able to check `KubeletConfig`
+configurations passed in as command flags and from other sources than `/etc/kubernetes/kubelet.conf`.
+It provides better accuracy in checking `KubeletConfig` configurations.
+
+## How does it work?
+To give an example on the CIS benchmark:
+The previous approach was to have `KubeletConfig` rules belong to `ocp-cis-node` profile. 
+When a user launches a scan using ScanSettingBinding:
+```yaml
+apiVersion: compliance.openshift.io/v1alpha1
+kind: ScanSettingBinding
+metadata:
+  name: cis
+  namespace: openshift-compliance
+profiles:
+- apiGroup: compliance.openshift.io/v1alpha1
+  kind: Profile
+  name: ocp4-cis
+- apiGroup: compliance.openshift.io/v1alpha1
+  kind: Profile
+  name: ocp4-cis-node
+settingsRef:
+  apiGroup: compliance.openshift.io/v1alpha1
+  kind: ScanSetting
+  name: default
+```
+The `kubelteConfig` rules checks is executed against node file `/etc/kubernetes/kubelet.conf`.
+Depending on the roles a user has set in `ScanSetting`, checks will be executed against nodes with the selected node type.
+And in `ComplianceCheckResult`, `kubelteConfig` rules will be shown as `ocp4-cis-node-<role>-kubelet-*`.
+
+In the new approach, `KubeletConfig` rules belong to `ocp4-cis` profile. Compliance Operator checks runtime `KubeletConfig`
+through Kube `Node/Proxy` object, and it uses variables `ocp-var-role-master` and `ocp-var-role-master` to determine the nodes
+it performs the check against. And in `ComplianceCheckResult`, `kubelteConfig` rules will be shown as `ocp4-cis-kubelet-*`.
+The scan only passes if all selected nodes pass the check.
+
+## How to use it?
+
+In a standard cluster that only has `master` and `worker` pool, there isn't much needed to be done, everything will work out of the box.
+
+### a cluster that uses sub-pools
+
+In a cluster that has `master`, `worker` and `infra` pool, If a user wants to check against all pools, they need to set the value of
```

Comment:
Since `infra` isn't a default pool, I'd prepend this sentence with `For example,`.